blog/templates/blog/post_detail.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'blog/base.html' %}
+
+{% block content %}
+    <div class="post">
+        {% if post.published_date %}
+            <div class="date">
+                {{ post.published_date }}
+            </div>
+        {% endif %}
+        {% if user.is_authenticated %}
+            <a class="btn btn-default"
+               href="{% url 'post_edit' pk=post.pk %}"><span
```

Comment:
Reformat this with the hierarchy in mind, not just do a new line on random space.

Reply:
@webknjaz Sorry?